### PR TITLE
✨ Archive export / import

### PR DESCRIPTION
**Pull request by pajowu**
This PR adds a new export mode that creates a "transcribee archive", i.e. a zip-file with the automerge document and a media file. When such a file is uploaded in the "New Document" page, the file is unzipped and imported.

This allows users to export a file from one transcribee instance to another.

### CODE REVIEW

File: backend/transcribee_backend/routers/document.py [hunk 1]
```diff
@@ -354,6 +355,58 @@ async def create_document(
     return document.as_api_document()
 
 
+@document_router.post("/import/")
+def replace_changes(
```
Comment (anuejn) [inline backend/transcribee_backend/routers/document.py:359]: maybe name this method import?

File: backend/transcribee_backend/routers/document.py [hunk 2]
```diff
@@ -354,6 +355,58 @@ async def create_document(
...
+    document_updates: List[UploadFile] = File(),
+    file: UploadFile = File(),
```
Comment (anuejn) [inline backend/transcribee_backend/routers/document.py:361]: what is file? -> better name?

Comment (anuejn) [reply backend/transcribee_backend/routers/document.py:361]: media_file?

File: frontend/src/editor/export/transcribee.tsx [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+              const zipFileBlob = await zipFileWriter.getData();
+              downloadBinaryAsFile(
+                `${outputNameBase}.transcribee`,
+                `application/octet-stream`,
+                zipFileBlob,
+              );
+
+              onClose();
+            } catch {
+              /* empty */
```
Comment (anuejn) [inline frontend/src/editor/export/transcribee.tsx:78]: maybe we at least want to log errors?

File: frontend/src/pages/new_document.tsx [hunk 4]
```diff
@@ -66,6 +85,9 @@ export function NewDocumentPage() {
     setValue('language', getLanguages(config.models, model)?.[0] || 'auto');
   }, [config, model]);
 
+  // Switch to import mode if a .transcribee-file is selected
```
Comment (anuejn) [inline frontend/src/pages/new_document.tsx:88]:
```suggestion
  // Switch to import mode if a .transcribee file is selected
```

File: frontend/src/pages/user_home.tsx [hunk 5]
```diff
@@ -35,75 +36,87 @@ export function UserHomePage() {
...
+        }}
+      />
+      {doc.name}
+    </LinkCard>
+  );
+}
+
+function NewDocumentCard() {
+  return (
+    <LinkCard title="create new document" to={`/new`}>
```
Comment (anuejn) [inline frontend/src/pages/user_home.tsx:87]: different capitalisation?

File: frontend/src/pages/new_document.tsx [hunk 6]
```diff
@@ -178,7 +232,7 @@ export function NewDocumentPage() {
                       }}
                       type="file"
                       className="opacity-0 absolute peer w-full"
-                      accept="audio/*,video/*"
+                      accept="audio/*,video/*,.transcribee"
```
Comment (anuejn) [inline frontend/src/pages/new_document.tsx:235]: I would like to have some text somewhere, that one can import files here

File: frontend/src/pages/new_document.tsx [hunk 7]
```diff
@@ -220,108 +274,122 @@ export function NewDocumentPage() {
...
-                        {lang}
-                      </option>
-                    ))}
-                  </Select>
-                </FormControl>
-              </div>
-            ) : (
-              <></>
-            )}
+            {!isImport && (
```
Comment (anuejn) [inline frontend/src/pages/new_document.tsx:277]: Can I have some alternative text when we are importing?

**Review by anuejn — CHANGES_REQUESTED** [review]

File: frontend/src/pages/user_home.tsx (hunk 5 above)
Comment (pajowu) [reply frontend/src/pages/user_home.tsx:87]: what do you mean?

File: frontend/src/pages/new_document.tsx (hunk 7 above)
Comment (pajowu) [reply frontend/src/pages/new_document.tsx:277]: What do you mean?

Comment (anuejn) [reply frontend/src/pages/new_document.tsx:275]: I would like to have some kind of feedback acknoledging that we are now importing an existing document. Also I would like to have some indicatino that this is possible here.

File: frontend/src/pages/user_home.tsx (hunk 5 above)
Comment (anuejn) [reply frontend/src/pages/user_home.tsx:87]: All the other texts we are using are using Title Case

Comment (pajowu) [reply frontend/src/pages/user_home.tsx:87]: Ah okay. I just kept them as they were. Fixed now

**Review by anuejn — APPROVED** [review]
Nice work :)

**Pull request merged**